tokio/src/runtime/builder.rs

```diff
@@ -82,7 +82,7 @@ pub struct Builder {
     pub(super) keep_alive: Option<Duration>,
 
     /// How many ticks before pulling a task from the global/remote queue?
```

Comment:
can this comment explain what it means for this value to be `None`, now that it's an `Option`?